fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/ApplicationAmbassador.java

```diff
@@ -299,17 +300,17 @@ protected void processInteraction(final Interaction interaction) throws Internal
         }
     }
 
-    private void process(final VehicleElectricityUpdates electricInformationMessage) {
+    private void process(final VehicleBatteryUpdates vehicleBatteryUpdates) {
         // schedule all updated vehicles
-        for (VehicleBatteryState vehicleBatteryState : electricInformationMessage.getUpdated()) {
-            final AbstractSimulationUnit simulationUnit = UnitSimulator.UnitSimulator.getUnitFromId(vehicleBatteryState.getName());
+        for (BatteryData batteryData : vehicleBatteryUpdates.getUpdated()) {
+            final AbstractSimulationUnit simulationUnit = UnitSimulator.UnitSimulator.getUnitFromId(batteryData.getName());
```

Comment:
With `batteryData.getName()` it is not really clear that it is the name of the vehicle the battery belongs to. We should rename this in `getBatteryOwner()` or the like.